### ISSUE TITLE
Add Bitrot Command

### ISSUE THREAD

**Issue by EnochPrime** [gluster/gluster-ansible-collection#18]
Glusterfs has the follow settings for bitrot:
- features.bitrot
- features.scrub
- features.scrub-freq
- features.scrub-throttle

However, these cannot be changed using the 'volume set' command and must be done with the 'volume bitrot' command.
> "error running gluster (/usr/sbin/gluster --mode=script volume set test features.bitrot on) command (rc=1): volume set: failed:  'gluster volume set <VOLNAME> features.bitrot' is invalid command. Use 'gluster volume bitrot <VOLNAME> {enable|disable}' instead.

**Comment by pkesavap** [comment]
@EnochPrime Could you let me know which version of gluster you are using?

**Comment by EnochPrime** [comment]
I don't recall which version I was using at the time, but I am currently using 8.4.

**Comment by pkesavap** [comment]
@EnochPrime Unfortunately since gluster is EOL, there is no active efforts happening to the repo.

That being said, if you are open to creating a  PR to include it, feel free to do so.

**Comment by EnochPrime** [comment]
> gluster is EOL

@pkesavap is gluster ceasing developement? I understand v8 should be reaching EOL in May.

**Comment by pkesavap** [comment]
Yes, I guess, v8 is reaching EOL around may. I suspect there won't be any additional efforts for RFE, especially to modules in this repo, unless there is a huge request for it.

**Comment by dg76** [comment]
@pkesavap You said "gluster is EOL". Can you tell me where I can find information about it? All I can find is that "Red Hat Gluster Storage" is EOL. I cannot find any hints about EOL of Gluster itself. On

https://www.gluster.org/release-schedule/

there is no EOL listed for Gluster 10 yet, which was released last February. Where can I find information that "gluster is EOL"?

Thank you very much!
